### PR TITLE
Update lint-staged

### PR DESCRIPTION
**Pull request by matamatanot**
https://github.com/okonet/lint-staged#migration

> From v10.0.0 onwards any new modifications to originally staged files will be automatically added to the commit. If your task previously contained a git add step, please remove this. The automatic behaviour ensures there are less race-conditions, since trying to run multiple git operations at the same time usually results in an error.

> From v10.0.0 onwards lint-staged requires Node.js version 10.13.0 or later.

### CODE REVIEW

**Comment by ijjk** [comment]
## Failing test suites 

**test/integration/size-limit/test/index.test.js** 
- Production response size > should not increase the overall response size of default build

<details>
<summary>Expand output</summary>

  ● Production response size › should not increase the overall response size of default build

    expect(received).toBeLessThanOrEqual(expected)

    Expected: <= 1024
    Received:    1076

      82 |     // These numbers are without gzip compression!
      83 |     const delta = responseSizesBytes - 257 * 1024
    > 84 |     expect(delta).toBeLessThanOrEqual(1024) // don't increase size more than 1kb
         |                   ^
      85 |     expect(delta).toBeGreaterThanOrEqual(-1024) // don't decrease size more than 1kb without updating target
      86 |   })
      87 | 

      at Object.<anonymous> (integration/size-limit/test/index.test.js:84:19)

</details>

**Comment by ijjk** [comment]
## Failing test suites 

**test/integration/size-limit/test/index.test.js** 
- Production response size > should not increase the overall response size of default build

<details>
<summary>Expand output</summary>

  ● Production response size › should not increase the overall response size of default build

    expect(received).toBeLessThanOrEqual(expected)

    Expected: <= 1024
    Received:    1076

      82 |     // These numbers are without gzip compression!
      83 |     const delta = responseSizesBytes - 257 * 1024
    > 84 |     expect(delta).toBeLessThanOrEqual(1024) // don't increase size more than 1kb
         |                   ^
      85 |     expect(delta).toBeGreaterThanOrEqual(-1024) // don't decrease size more than 1kb without updating target
      86 |   })
      87 | 

      at Object.<anonymous> (integration/size-limit/test/index.test.js:84:19)

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| buildDuration | 12s | 11.8s | -240ms |
| nodeModulesSize | 55.2 MB | 55.2 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.037 | 1.992 | -0.04 |
| / avg req/sec | 1227.17 | 1255.08 | ⚠️ +27.91 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.02 | 1.03 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 2449.81 | 2428.13 | -21.68 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.27 kB | 6.27 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..ac5e.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.82 kB | 4.82 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.88 kB | 6.88 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 51.5 kB | 51.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.2 kB | 26.2 kB | ✓ |
| Overall change | 26.2 kB | 26.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.12 kB | 3.12 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.85 kB | 7.85 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| index.html gzip | 927 B | 927 B | ✓ |
| link.html gzip | 937 B | 937 B | ✓ |
| withRouter.html gzip | 925 B | 925 B | ✓ |
| Overall change | 2.79 kB | 2.79 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| buildDuration | 12.4s | 12.7s | ⚠️ +236ms |
| nodeModulesSize | 55.2 MB | 55.2 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.27 kB | 6.27 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..ac5e.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.82 kB | 4.82 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.88 kB | 6.88 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 51.5 kB | 51.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.2 kB | 26.2 kB | ✓ |
| Overall change | 26.2 kB | 26.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.12 kB | 3.12 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.85 kB | 7.85 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _error.js | 558 kB | 558 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.81 kB | 3.81 kB | ✓ |
| index.js | 558 kB | 558 kB | ✓ |
| link.js | 595 kB | 595 kB | ✓ |
| routerDirect.js | 587 kB | 587 kB | ✓ |
| withRouter.js | 587 kB | 587 kB | ✓ |
| Overall change | 2.89 MB | 2.89 MB | ✓ |

</details>

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| buildDuration | 12.2s | 12.3s | ⚠️ +87ms |
| nodeModulesSize | 56.4 MB | 56.4 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.095 | 2.102 | ⚠️ +0.01 |
| / avg req/sec | 1193.58 | 1189.39 | -4.19 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.349 | 1.413 | ⚠️ +0.06 |
| /error-in-render avg req/sec | 1853.26 | 1769.58 | -83.68 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.28 kB | 6.28 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5232.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39.2 kB | 39.2 kB | ✓ |
| Overall change | 57 kB | 57 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.82 kB | 4.82 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.88 kB | 6.88 kB | ✓ |
| framework.HA..dule.js gzip | 39.2 kB | 39.2 kB | ✓ |
| Overall change | 51.7 kB | 51.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.13 kB | 3.13 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| index.html gzip | 927 B | 927 B | ✓ |
| link.html gzip | 936 B | 936 B | ✓ |
| withRouter.html gzip | 925 B | 925 B | ✓ |
| Overall change | 2.79 kB | 2.79 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| buildDuration | 13s | 13.1s | ⚠️ +12ms |
| nodeModulesSize | 56.4 MB | 56.4 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.28 kB | 6.28 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5232.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39.2 kB | 39.2 kB | ✓ |
| Overall change | 57 kB | 57 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.82 kB | 4.82 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.88 kB | 6.88 kB | ✓ |
| framework.HA..dule.js gzip | 39.2 kB | 39.2 kB | ✓ |
| Overall change | 51.7 kB | 51.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.13 kB | 3.13 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | matamatanot/next.js update-lint-staged | Change |
| - | - | - | - |
| _error.js | 853 kB | 853 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.81 kB | 3.81 kB | ✓ |
| index.js | 853 kB | 853 kB | ✓ |
| link.js | 891 kB | 891 kB | ✓ |
| routerDirect.js | 883 kB | 883 kB | ✓ |
| withRouter.js | 883 kB | 883 kB | ✓ |
| Overall change | 4.37 MB | 4.37 MB | ✓ |

</details>

</details>

File: lint-staged.config.js [hunk 1]
```diff
@@ -11,16 +11,12 @@ module.exports = {
...
   },
   '**/*.{json,md,mdx,css,html,yml,yaml,scss}': filenames => {
     const escapedFileNames = filenames
       .map(filename => `"${isWin ? filename : escape([filename])}"`)
       .join(' ')
-    return [
-      `prettier --with-node-modules --ignore-path .prettierignore_staged --write ${escapedFileNames}`,
-      `git add ${escapedFileNames}`,
-    ]
+    return `prettier --with-node-modules --ignore-path .prettierignore_staged --write ${escapedFileNames}`
```
Comment (Timer) [inline lint-staged.config.js:20]:
```suggestion
    return [`prettier --with-node-modules --ignore-path .prettierignore_staged --write ${escapedFileNames}`]
```

**Review by Timer — APPROVED** [review]

**Comment by Timer** [comment]
Thanks!

**Pull request merged**